checks/51-rpmlint

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+RPMLINT_CHROOT="$BUILD_ROOT/usr/lib/rpmlint-image"
+WORKDIR=$RPMLINT_CHROOT/RPM
+SOURCES="$BUILD_ROOT/home/abuild/rpmbuild/SOURCES"
+
+find $BUILD_ROOT/home/abuild/rpmbuild/RPMS -type f -name '*.rpm' -exec cp {} $WORKDIR \;
```

Comment:
Path shouldn't be hardcoded: https://github.com/openSUSE/post-build-checks/blob/d341ab0ee0ea0b174a510858674162129ddd30f0/checks/00-check-install-rpms#L7

Reply:
Looking that `TOPDIR` is defined in the checks as `TOPDIR=/usr/src/packages`.
How is that related to `/home/abuild/rpmbuild/` which is a location I see when I build a package locally?

Reply:
It's overwritten in line 5, maybe they're copied into that location at some point.

Reply:
You are right! Works for me.